crates/harp/src/size.rs

```diff
@@ -611,4 +619,42 @@ mod tests {
             assert!(size != 0)
         })
     }
+
+    // This is a regression test for:
+    // - https://github.com/posit-dev/positron/issues/4686
+    // - https://github.com/posit-dev/positron/issues/4741
+    // In both cases the R session crashed because we tried to call `CAR`
+    // in a immediate binding. More info in:
+    // https://github.com/posit-dev/positron/issues/4686#issuecomment-2352427239
+    #[test]
+    fn test_shinytest2_size() {
```

Comment:
hmm... I'd rather not have integration tests of this kind here.
I would prefer a unit test.

What do you think @DavisVaughan?

Reply:
Ok! I'm happy to remove this test. I unfortunatelly can't find a way of constructing these inlined values in a simple way.